tools/eslint-rules/project.json

```diff
@@ -10,5 +10,6 @@
         "jestConfig": "tools/eslint-rules/jest.config.ts"
       }
     }
-  }
+  },
+  "tags": ["tools"]
```

Comment:
@Hotell I'm unsure about this one, do we need the "tools" tag here?

Reply:
yup, we need to normalize ( add missing tags where applicable )